packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -484,6 +484,11 @@ export class TeamsAppSolution implements Solution {
     if (canProvision.isErr()) {
       return canProvision;
     }
+
+    if (!ctx.config.get(GLOBAL_CONFIG)) {
+      ctx.config.set(GLOBAL_CONFIG, new ConfigMap());
+    }
```

Comment:
Suppose the solution config is checked and loaded in the context loader. How to about move it to the `envInfoLoader`?

Reply:
updated and move the logic while loading the env profile.